src/components/Navigation/Navbar.js

```diff
@@ -32,6 +32,11 @@ function Navbar() {
                 Sign In
               </Link>
             </li>
+            <li className="nav-item">
+              <Link to="/single-match-popup" className="nav-links" onClick={closeMobileMenu}>
```

Comment:
I think this should be changed to @knguyen99 's match page. I only added '/single-match-popup' so that I could test my match popup component. It's not actually supposed to be accessed by the user.